packages/create-block-quick-start-template/block-templates/README.md.mustache

```diff
@@ -0,0 +1,16 @@
+# Interactive Block
+
+> **Warning**
+> **This block requires Gutenberg 16.2 or superior to work**. The Interactivity API is, at the moment, not part of WordPress Core as it is still very experimental, and very likely to change. 
+
+> **Note**
+> This block uses the API shared at [Proposal: The Interactivity API – A better developer experience in building interactive blocks](https://make.wordpress.org/core/2023/03/30/proposal-the-interactivity-api-a-better-developer-experience-in-building-interactive-blocks/). 
+
+{{#isBasicVariant}}
```

Comment:
We should probably remove this as you don't have any variants defined so this will never render

Reply:
Yeah, this Readme should not have been included. Copy and paste error 🤦‍♂️ All fixed